### PR TITLE
Add test for processes table to verify mem/cpu units

### PR DESCRIPTION
**Pull request by theopolis**
Assure new platforms include a `total_size` and `resident_size` for memory reporting in bytes units. Assure new platforms include a `user_time` and `system_time` to report CPU cycles uses in seconds units.

### CODE REVIEW

File: osquery/tables/system/tests/system_tables_tests.cpp [hunk 1]
```diff
@@ -15,35 +15,33 @@
...
-  QueryContext context;
-  auto result = genOSVersion(context);
-  EXPECT_EQ(result.size(), 1U);
+  auto results = SQL("select * from os_version");
+
+  // Issue #2564: There is no os_version on Windows.
+  if (!isPlatform(PlatformType::TYPE_WINDOWS)) {
+    EXPECT_EQ(results.rows().size(), 1U);
 
-  // Make sure major and minor contain data (a missing value of -1 is an error).
```
Comment (PoppySeedPlehzr) [inline osquery/tables/system/tests/system_tables_tests.cpp]: Any reason why we dropped the `contain` here?

**Comment by osqueryer** [comment]
:-1: The commit b3b83314ae2c27c7a6ea21726ab04dd77d3b7ff2 (Job results: [3431](https://jenkins.osquery.io/job/osqueryPullRequestBuild/3431/)) failed one or more tests (**OS X/Linux**).

**Comment by osqueryer** [comment]
:-1: The commit 509eb471092d0184b8a0d3eb929cc69e1b9d466f (Job results: [3433](https://jenkins.osquery.io/job/osqueryPullRequestBuild/3433/)) failed one or more tests (**OS X/Linux**).

**Comment by osqueryer** [comment]
:-1: The commit 509eb471092d0184b8a0d3eb929cc69e1b9d466f (Job results: [437](https://jenkins.osquery.io/job/osqueryPullRequestBuild/437/)) failed one or more tests (**Windows**).

**Pull request merged**